site/pages/docs/ref/add-cal/add-cal-fr.hbs

```diff
@@ -0,0 +1,111 @@
+		</tr>
+		<tr>
+			<td>jQuery Event</td>
+			<td><code>wb-ready.wb-addcal</code></td>
+			<td>Triggered automatically after the plugin initializes.</td>
+			<td>Used to identify when and where the plugin initializes (target of the event).
+				<pre><code>$( document ).on( "wb-ready.wb-addcal", ".wb-addcal", function( event ) {
+	});</code></pre>
+				<pre><code>$elm.on( "wb-ready.wb-addcal", function( event ) {
+	});</code></pre>
+			</td>
+		</tr>
+	</table>
+
+	<h3>Configuration</h3>
+	<p>Not applicable for this plugin.</p>
+
+	<h3>User interface (Template)</h3>
+	<p>(Provisional)</p>
+
+	<p>The Schema.org <code>Event</code> type noted in RDFa is required when implementing this add to calendar widget. It identifies an event happeing at a certain time and location. The followings are Schema.org types and properties:</p>
+	<dl>
+		<dt><code>name</code> (Event)</dt>
+		<dd>Required - Short text that defines the event title.</dd>
+		<dt><code>startDate</code> (Event)</dt>
+		<dd>Required - Specifies the start date and or time of the event. The following pattern must be used: "YYY-MM-DD HH:MMZ"</dd>
+		<dt><code>endDate</code> (Event)</dt>
+		<dd>Required - Specifies the start date and or time of the event. The following pattern must be used: "YYY-MM-DD HH:MMZ"</dd>
+		<dt><code>location</code> (Event)</dt>
+		<dd>Recommended - Short text that Identfies the place of the event or adding type of Place to provide a detailed address.</dd>
+		<dt><code>name</code> (Place)</dt>
+		<dd>Optional - Short text that ndentifies the name of a location for the event.</dd>
+		<dt><code>address</code> (PostalAddress)</dt>
+		<dd>Optional - Use with type of PostalAddress to initiate a detailed location of the event.</dd>
+		<dt><code>streetAddress</code> (PostalAddress)</dt>
+		<dd>Optional - Text that identifies the street address of the event.</dd>
+		<dt><code>addressLocality</code> (PostalAddress)</dt>
+		<dd>Optional - Text that specifies the locatily (city) of the event.</dd>
+		<dt><code>addressRegion</code> (PostalAddress)</dt>
+		<dd>Optional - Text that specifies the region (province) of the event.</dd>
+		<dt><code>description</code> (Event)</dt>
+		<dd>Optional - Short text that provides a brief description of the event.</dd>
+	</dl>
+
+	<p>The HTML element <code>time</code> must be used when specifying a date and or time in machine readable format or adding the <code>datetime</code> attribute when using text. The following pattern must be applied when using the machine readable format in the time element or as value for the datetime attribute: "YYYY-MM-DD HH:MMZ".</p>
+
+	<p>The propery "datemodified" must be present in the web page.</p>
+
+	<dd>Required - HTML element must be used with any date and or time.</dd>
+		<dt><code>datetime</code> (time)</dt>
+		<dd>Optional - HTML attribute from 'time' element use to specify date and or time in machine readable format. The following pattern must be used: "YYY-MM-DD HH:MMZ"</dd>
```

Comment:
Please remove this, and instead integrate that info with the `startDate` and `endDate`. Like my previous comment